src/_tutorials/server/cmdline.md

```diff
@@ -93,13 +93,17 @@ brief description of key app features, click the highlighted code below.
 
 Here's an example of running the app from the command line:
 
-```terminal
-$ dart dcat.dart -n quote.txt
-1 Be yourself. Everyone else is taken. -Oscar Wilde
-2 Don't cry because it's over, smile because it happened. -Dr. Seuss
-3 You only live once, but if you do it right, once is enough. -Mae West
-...
-```
+ 1. Download the file called [quote.txt]({{site.repo.this}}/blob/master/examples/misc/test_data/quote.txt){:target="_blank"} and save it.
+ 
+ 2. In the directory that contains the file quote.txt and dcat.dart, run the program:
+ 
+    ```terminal
+     $ dart dcat.dart -n quote.txt
+     1 Be yourself. Everyone else is taken. -Oscar Wilde
+     2 Don't cry because it's over, smile because it happened. -Dr. Seuss
+     3 You only live once, but if you do it right, once is enough. -Mae West
+     ...
+     ```
```

Comment:
Hi. Thanks for the PR!

Making the download of `quote.txt` a _required_ step feels a little to heavy and unnecessarily constraining. How about simply replacing

> Here's an example of running the app from the command line:

with something like this:

> You can run the app from the command line over the `quote.txt` file (which you can [download from this repo](#put-the-right-link-here)) or any other text file, as shown here:

Good job on the encoding of the link. I'd suggest appending `?raw=1` to the end of that link so that the reader is presented with the raw content immediately.

WDYT? @kwalrath?